client/src/pages/Registration/RegistrationFields.jsx

```diff
@@ -96,7 +67,7 @@ export const fields = {
         'Civil',
         'Computer',
         'Electrical',
-        'Engineering',
+        'Engineering Science',
         'Industrial',
         'Materials',
         'Mechanical',
```

Comment:
Again, whoever is working on #131 will need to ensure that these match with the options provided